spec/controllers/users_controller_spec.rb

```diff
@@ -22,7 +22,40 @@
         stub_api_v2_request("/users/#{user.id}/accept_transition_screen", {}, :patch, 200)
       end
 
-      it "redirects to providers index" do
+      it "redirects to the rollover screen" do
+        get :accept_transition_info
+        expect(response).to redirect_to(rollover_path)
+      end
+    end
+
+    context "with client error" do
+      before do
+        stub_api_v2_request("/users/#{user.id}/accept_transition_screen", {}, :patch, 400)
+      end
+
+      it "redirects to rollover screen" do
+        get :accept_transition_info
+        expect(Raven).to have_received(:capture_exception).with(instance_of(JsonApiClient::Errors::ClientError))
+        expect(response).to redirect_to(rollover_path)
+      end
+    end
+  end
+
+  describe 'GET #accept_transition_info when rollover is disabled' do
+    before do
+      Settings.rollover = false
```

Comment:
Must be a better way to do this.

Reply:
https://github.com/rspec/rspec-mocks#method-stubs

Something like

```ruby
allow(Settings).to receive(:rollover).and_return(false)
```